centos-7-minimal.ks

```diff
@@ -65,10 +66,29 @@ shim
 LANG="en_US"
 echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
 
+# Make sure to properly handle the ssh keys passed to us via docker-machine
+cat > /etc/rc.d/init.d/copy-ssh << EOF
```

Comment:
@praveenkumar @LalatenduMohanty, so here is what I meant we can focus just on the ssh part for now.
This changes actually create the init script _copy-ssh_ and put it at the right location in the VM.

Also, if I log onto the VM and run the script, it does what I want. I creates the docker user and extracts the ssh keys. Unfortunately, it does not happen at bootstrap. When I log in after startup, the docker user is not created yet.